### ISSUE TITLE
rMATS output files

### ISSUE THREAD

**Issue by NorahAlghamdi1408** [Xinglab/rmats-turbo#234]
Hello, I am using rMATS and my BAM files were processed without any issue but the output has only header except some GTF files 

[alghn0a@login509-02-l bams2]$  python $rMATS/rmats.py --b1 trynova1.txt --b2 trynova2.txt --gtf Mus_musculus.GRCm39.107.gtf --readLength 50  --od Nova1_AS_out
There are 56748 distinct gene ID in the gtf file
There are 149478 distinct transcript ID in the gtf file
There are 35106 one-transcript genes in the gtf file
There are 869027 exons in the gtf file
There are 28190 one-exon transcripts in the gtf file
There are 22687 one-transcript genes with only one exon in the transcript
Average number of transcripts per gene is 2.634066
Average number of exons per transcript is 5.813745
Average number of exons per transcript excluding one-exon tx is 6.932565
Average number of gene per geneGroup is 7.513409

==========
Done processing each gene from dictionary to compile AS events
Found 20986 exon skipping events
Found 1141 exon MX events
Found 9310 alt SS events
There are 5866 alt 3 SS events and 3444 alt 5 SS events.
Found 4505 RI events
==========

Running the statistical part.
The statistical part is done.
Done.

Does anyone know how to solve it? 
Thank you

**Comment by EricKutschera** [comment]
My guess is that all or almost all reads are being filtered out. The recent versions of rMATS will output a summary of how many reads were filtered. See this related post: https://github.com/Xinglab/rmats-turbo/issues/89

**Comment by NorahAlghamdi1408** [comment]
Yes, you are right, I installed the recent version and I got this result, I don't know why no reads are used 

my command is 
rmats.py --b1 trynova1.txt --b2 trynova2.txt --gtf Mus_musculus.GRCm39.107.gtf --readLength 50  --od Nova1_AS_out2 --tmp fold


 [alghn0a@login509-02-r tmporary_file]$ cat  2022-11-02-11_32_59_326463_read_outcomes_by_bam.txt
BW_1-1_S2_L002_R2_001.fastqAligned.sortedByCoord.out.bam
USED: 0
NOT_PAIRED: 32326318
NOT_NH_1: 0
NOT_EXPECTED_CIGAR: 0
NOT_EXPECTED_READ_LENGTH: 0
NOT_EXPECTED_STRAND: 0
EXON_NOT_MATCHED_TO_ANNOTATION: 0
JUNCTION_NOT_MATCHED_TO_ANNOTATION: 0
CLIPPED: 4180470
TOTAL_FOR_BAM: 36506788
BW_SST-WT_Sham_S17_L004_R2_001.fastqAligned.sortedByCoord.out.bam
USED: 0
NOT_PAIRED: 29895257
NOT_NH_1: 0
NOT_EXPECTED_CIGAR: 0
NOT_EXPECTED_READ_LENGTH: 0
NOT_EXPECTED_STRAND: 0
EXON_NOT_MATCHED_TO_ANNOTATION: 0
JUNCTION_NOT_MATCHED_TO_ANNOTATION: 0
CLIPPED: 9760199
TOTAL_FOR_BAM: 39655456
[alghn0a@login509-02-r tmporary_file]$


Do you have any suggestions please?

**Comment by EricKutschera** [comment]
Most of the reads are being filtered for NOT_PAIRED. If your reads are not paired then you can run with `-t single`. Here is a related post: https://github.com/Xinglab/rmats-turbo/issues/153
